src/main.cpp

```diff
@@ -1,13 +1,43 @@
 #include <string>
-#include <iostream>     //  std::cerr, std::cout, std::endl
-#include <filesystem>   //  exists, create_directories
+#include <iostream>    // std::cerr, std::cout, std::endl
+
+#ifdef _WIN32
+#include <windows.h>
+#else
+#include <sys/stat.h>  // mkdir
+#include <unistd.h>    // access
+#endif
 
 #include "RedatamDatabase.hpp"
 
-using std::string, std::cerr, std::cout, std::endl;
-namespace fs = std::filesystem;
+using std::cerr;
+using std::cout;
+using std::endl;
+using std::string;
+
+#ifdef _WIN32
+bool exists(const string& path)
+{
+    DWORD attr = GetFileAttributes(path.c_str());
+    return (attr != INVALID_FILE_ATTRIBUTES);
+}
+
+bool create_directories(const string& path)
+{
+    return CreateDirectory(path.c_str(), NULL) ||
+        GetLastError() == ERROR_ALREADY_EXISTS;
+}
+#else
+bool exists(const string& path) { return access(path.c_str(), F_OK) != -1; }
 
-int main(int argc, char *argv[])
+bool create_directories(const string& path)
+{
+    mode_t mode = 0755;
```

Comment:
@pachadotdev please use or'd flags insted of a magic number
i.e. 
S_IRUSR || S_IWUSR || S_IXUSR || S_IRGRP || S_IXGRP || S_IROTH || S_IXOTH